relayer/workers/parachaincommitmentrelayer/catchup.go

```diff
@@ -185,7 +187,7 @@ func (li *Listener) searchForLostCommitments(lastBlockNumber uint64, basicNonceT
 	return nil
```

Comment:
Lost commitments are not returned/processed here, will that be part of this PR? Or do we have a task for that?

Reply:
yep in progress in this PR